tests/foreman/sys/test_pulp3_filesystem.py

```diff
@@ -26,13 +28,18 @@ def test_selinux_status(target_sat):
     :id: 43218070-ac5e-4679-b74a-3e2bcb497a0a
 
     :expectedresults: SELinux is enabled and there are no denials
+
+    :customerscenario: true
+
+    :BZ: 2131031
     """
     # check SELinux is enabled
     result = target_sat.execute('getenforce')
     assert 'Enforcing' in result.stdout
     # check there are no SELinux denials
-    result = target_sat.execute('ausearch --input-logs -m avc -ts today --raw')
-    assert result.status == 1
+    if target_sat.os_version.major == 7 or not is_open('BZ:2131031'):
```

Comment:
For current satellite we will never have `target_sat.os_version.major == 7`. Maybe only add that for the cherry pick?